src/vaadin-combo-box-data-provider-mixin.html

```diff
@@ -257,5 +261,27 @@
       }
     }
 
+    /**
+     * This method cleans up the page callbacks which refers to the
+     * non-existing pages, i.e. which item indexes are greater than the
+     * changed size.
+     * This case is basically happens when:
+     * 1. Users scroll fast to the bottom and combo box generates the
+     * redundant page request/callback
+     * 2. Server side uses undefined size lazy loading and suddenly reaches
+     * the exact size which is on the range edge
+     * (for default page size = 50, it will be 100, 200, 300, ...).
+     * @param size the new size of items
+     * @private
+     */
+    _flushPendingRequests(size) {
+      const lastPage = Math.ceil(size / this.pageSize);
+      for (const page of Object.keys(this._pendingRequests)) {
```

Comment:
Please do not use `for .. of` as it will break in Vaadin 14.x bower mode.

See more context here: https://github.com/vaadin/flow/issues/5164#issuecomment-469146529

Reply:
Thanks! Will fix that

Reply:
done